x-pack/plugins/files/server/blob_storage_service/adapters/es/es.ts

```diff
@@ -5,25 +5,30 @@
  * 2.0.
  */
 
+import { Readable, pipeline as _pipeline } from 'stream';
+import { promisify } from 'util';
 import type { ElasticsearchClient, Logger } from '@kbn/core/server';
 import { errors } from '@elastic/elasticsearch';
-import { ReadStream } from 'fs';
-
 import type { BlobStorage } from '../../types';
-
 import { mappings } from './mappings';
+import { getWritableContentStream, getReadableContentStream } from './content_stream';
 
-const BLOB_STORAGE_SYSTEM_INDEX_NAME = '.kibana_blob_storage';
+export const BLOB_STORAGE_SYSTEM_INDEX_NAME = '.kibana_blob_storage';
```

Comment:
Do we need to expose this information (index name) to anyone? Would be nice to keep it implementation detail of this adapter.

Reply:
True, this was a convenience thing with the current tests so that I could assert against `esClient.indices.exists` checks. I thought it was OK to do this as long as it is not exported via the `index.ts` file of this adapter. WDYT?

If you are happy with that I can also add a comment about that.

Reply:
Sounds good.